signer/src/storage/postgres.rs

```diff
@@ -2268,6 +2268,35 @@ impl super::DbRead for PgStore {
         .await
         .map_err(Error::SqlxQuery)
     }
+
+    async fn get_valid_withdrawal_outputs(
+        &self,
+        id: &model::QualifiedRequestId,
+    ) -> Result<Vec<model::BitcoinWithdrawalOutput>, Error> {
+        sqlx::query_as::<_, model::BitcoinWithdrawalOutput>(
+            r#"
+            SELECT
+                bwo.bitcoin_txid
+              , bwo.bitcoin_chain_tip
+              , bwo.output_index
+              , bwo.request_id
+              , bwo.stacks_txid
+              , bwo.stacks_block_hash
+              , bwo.validation_result
+              , bwo.is_valid_tx
+            FROM sbtc_signer.bitcoin_withdrawals_outputs AS bwo
+            WHERE bwo.request_id = $1
+              AND bwo.stacks_block_hash = $2
+              AND bwo.validation_result = 'ok'
+              AND bwo.is_valid_tx = true
```

Comment:
Hmmm, I'm not sure if we want to filter on these values. It's possible that the current signer disapproves of the transaction for some reason but the consensus majority think that everything is fine. We'd want this signer to scan the mempool for that transaction as well.

Reply:
I was thinking it was fine since worst case the coordinator will go ahead with the signing round but the signers will prevent it (as they will need to do the same check, but they would have validated the outputs), but since it's cheap to check for the tx and we know the amount of them is kinda bounded under reasonable conditions (~18 + the number of consecutive blocks the signers RBFs without a confirmation) I agree that erring on checking more of them is better (and removed that check in [bc3ea8f](https://github.com/stacks-network/sbtc/pull/1398/commits/bc3ea8f216f14087f02b6c731ad7348e57e9d1bf)).

Reply:
Ahhh I see. Yeah it probably doesn't matter since the other signers will probably catch this. But yeah it doesn't hurt much to do.